homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -176,6 +176,7 @@ def _convert_forecast(self, entry):
             ATTR_API_FORECAST_PRESSURE: entry.pressure.get("press"),
             ATTR_API_FORECAST_WIND_SPEED: entry.wind().get("speed"),
             ATTR_API_FORECAST_WIND_BEARING: entry.wind().get("deg"),
+            ATTR_API_WEATHER: entry.detailed_status,
```

Comment:
It's not allowed to add keys not already defined in the Forecast dict.
See [developer docs](https://developers.home-assistant.io/docs/core/entity/weather#forecast-data) for what can be added.

This seems to be more fit for a `sensor` entity rather than extending the forecast

Reply:
I'd like to use the human readable weather for each hour. Having separate sensors for every hour isn't something that would fit.

I'll check if it's possible to add an attribute to the global weather component, but that will take some time.

I will keep this as a draft for now.

Reply:
Another option is perhaps to get it using a service call if you think that could be a fit.
I'm going to close this as to have it within the forecasts require an architectural decision.
Feel free to reopen the PR if a decision has been made to approve this addition or in the case you want to remake this PR into something approved (sensor, service call etc.)
Thanks 👍

Reply:
I don't think a service would be a good fit.

In the developer docs is this line:
> This means that the weather platforms don't need to support languages.

So a modification to the core also isn't happening. I will convert my modifications to a patch that can be applied with a HACS repo.